### PR TITLE
Add topic id to describe topics response

### PR DESCRIPTION
**Pull request by pranavrth**

### CODE REVIEW

File: src/Confluent.Kafka/Uuid.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+        /// <summary>
+        ///     Most significant 64 bits of the 128 bits UUID.
+        /// </summary>
+        public Offset MostSignificantBits { get; }
+
+        /// <summary>
+        ///     Most significant 64 bits of the 128 bits UUID.
+        /// </summary>
+        public Offset LeasttSignificantBits { get; }
```
Comment (anchitj) [inline src/Confluent.Kafka/Uuid.cs:58]: typo

File: src/Confluent.Kafka/Uuid.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        /// </param>
+        /// <param name="leastSignificantBits">
+        ///     Least significant 64 bits of the 128 bits UUID.
+        /// </param>
+        public Uuid(long mostSignificantBits, long leastSignificantBits)
+        {
+            MostSignificantBits = mostSignificantBits;
+            LeasttSignificantBits = leastSignificantBits;
+            IntPtr cUuid = Librdkafka.uuid_new(mostSignificantBits, leastSignificantBits);
+            IntPtr cBase64str = Librdkafka.uuid_base64str(cUuid);
```
Comment (anchitj) [inline src/Confluent.Kafka/Uuid.cs:45]: We should add null checks here, and do cleanup if null occurs.

**Review by anchitj — CHANGES_REQUESTED** [review]

File: src/Confluent.Kafka/AdminClient.cs [hunk 3]
```diff
@@ -509,6 +511,19 @@ private DescribeTopicsReport extractDescribeTopicsResults(IntPtr resultPtr)
             }).ToList();
             return result;
         }
+
+        private Uuid extractUuid(IntPtr uuidPtr)
+        {
+            if(uuidPtr == IntPtr.Zero)
```
Comment (emasab) [inline src/Confluent.Kafka/AdminClient.cs:517]:
```suggestion
            if (uuidPtr == IntPtr.Zero)
```

File: src/Confluent.Kafka/Impl/NativeMethods/NativeMethods_Centos6.cs [hunk 4]
```diff
@@ -315,6 +315,27 @@ internal class NativeMethods_Centos6
...
+        internal static extern /* rd_kafka_Uuid_t * */IntPtr rd_kafka_Uuid_new(
+                long most_significant_bits,
+                long least_significant_bits
+        );
+
+        [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern /* char * */IntPtr rd_kafka_Uuid_base64str(IntPtr uuid);
+
+        [DllImport(DllName, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern IntPtr rd_kafka_TopicDescription_topic_id(IntPtr topicdesc);
```
Comment (emasab) [inline src/Confluent.Kafka/Impl/NativeMethods/NativeMethods_Centos6.cs:328]: Move this after `rd_kafka_TopicDescription_name` as in `NativeMethods.cs`

File: src/Confluent.Kafka/Impl/LibRdKafka.cs [hunk 5]
```diff
@@ -1029,6 +1035,22 @@ internal static IntPtr consumer_group_metadata_read(out IntPtr cgmd, byte[] data
         private static Func<IntPtr, IntPtr> _memberid;
         internal static IntPtr memberid(IntPtr rk) => _memberid(rk);
 
+        private static Func<long, long, IntPtr> _uuid_new;
```
Comment (emasab) [inline src/Confluent.Kafka/Impl/LibRdKafka.cs:1038]: These fields and methods should start with Uuid, as we just remove the rd_kafka prefix here.
```suggestion
        private static Func<long, long, IntPtr> _Uuid_new;
```

File: src/Confluent.Kafka/Uuid.cs (hunk 2 above)
Comment (emasab) [reply src/Confluent.Kafka/Uuid.cs:45]: With latest changes conversion to string is always available even without OpenSSL

**Review by emasab — CHANGES_REQUESTED** [review]

File: src/Confluent.Kafka/Uuid.cs (hunk 2 above)
Comment (emasab) [reply src/Confluent.Kafka/Uuid.cs:45]: With latest changes conversion to string is always available even without OpenSSL

File: src/Confluent.Kafka/Admin/TopicDescription.cs [hunk 6]
```diff
@@ -32,6 +32,11 @@ public class TopicDescription
         /// </summary>
         public string Name { get; set; }
 
+        /// <summary>
+        ///     The topic Id.
+        /// </summary>
+        public Uuid TopicId {get; set; }
```
Comment (emasab) [inline src/Confluent.Kafka/Admin/TopicDescription.cs:38]: We should add the TopicId to the JSON returned by ToString

**Review by emasab — APPROVED** [review]
LGTM!

**Comment by emasab** [comment]
ok solved

**Review by anchitj — APPROVED** [review]

**Comment by emasab** [comment]
Thanks @pranavrth , seen last commit

**Pull request merged**